### ISSUE TITLE
Display compiler versions (and, optionally, to switch between them)

### ISSUE THREAD

**Issue by novusnota** [tact-lang/web-ide#20]
### Observed behaviour

At the moment, one has to do a set of heuristics to determine which compiler version of FunC or Tact the project is using.

For example, to check if the compiler supports Tact v1.2.0, you may try to use `+=` operator. And if that doesn't work, the version must be lower than 1.2.0.

Repeat this process to find the actual version used at the moment (or look into Nujan source code to find out faster, though still a bit impractical :)

### Expected behavior or possible solutions

1. Display the currently used compiler version — it may be hidden under the cog icon of settings on the lower left.
2. Optionally, make it possible to switch between a pre-defined set of FunC/Tact versions.

**Comment by anton-trunov** [comment]
@rahulyadav-57 We should definitely have the functionality to switch between Tact versions: both released ones (should support at least two latest minor releases) and the version from GitHub.

**Comment by rahulyadav-57** [comment]
Absolutely, adding the ability to switch between Tact versions would provide great flexibility.
